builder/bdwgc.go

```diff
@@ -30,6 +30,10 @@ var BoehmGC = Library{
 			// Use a minimal environment.
 			"-DNO_MSGBOX_ON_ERROR", // don't call MessageBoxA on Windows
 			"-DDONT_USE_ATEXIT",
+			"-DNO_CLOCK",           // don't use system clock
+			"-DNO_DEBUGGING",       // reduce code size
+			"-DGC_NO_FINALIZATION", // finalization is not used at the moment
+			"-DNO_GETENV",          // smaller binary, more predictable configuration
```

Comment:
"-DGC_DISABLE_INCREMENTAL" should also help in decreasing code size

Reply:
Also, consider `-D SMALL_CONFIG` (which implies `GC_DISABLE_INCREMENTAL`) and `-D IGNORE_DYNAMIC_LOADING`.

Reply:
I already use `"-DGC_DISABLE_INCREMENTAL"`, see a few lines above.
(I might try to enable incremental GC at some point, but I'll need to be careful around handling SIGSEGV etc so it's disabled at the moment).

`-D SMALL_CONFIG` docs say:

> Tries to tune the collector for small heap sizes, usually causing it to use less space in such situations. Incremental collection no longer works in this case.

Some people might want to use a large heap size, I'd like to keep that ability. This is a generic configuration, not for a specific program so would like to avoid tuning for a specific configuration.

Reply:
You could define -D LARGE_CONFIG in conjunction with -D SMALL_CONFIG. It might seem confllusing a bit:)

Reply:
Another available option (with the latest bdwgc master): `-D NO_FIND_LEAK`